.travis.yml

```diff
@@ -3,7 +3,7 @@ language: ruby
 sudo: false
 cache: bundler
 rvm:
-- 2.4.5
+- 2.5.7
```

Comment:
@slemrmartin can you add 2.6.5 also?

Reply:
@agrare are the pods running on 2.6?

Reply:
They're on 2.5.5 but it would be nice to know that it passes on 2.6 also for when the base image is updated.